src/main/java/games/strategy/engine/framework/systemcheck/LocalSystemChecker.java

```diff
@@ -37,7 +36,7 @@ private static SystemCheck defaultNetworkCheck() {
         urlConnection.setConnectTimeout(connectTimeoutInMilliseconds);
         urlConnection.connect();
       } catch (final Exception e) {
-        Throwables.propagate(e);
+        throw new RuntimeException(e);
```

Comment:
I was going to ask why this wasn't converted to

```java
throwIfUnchecked(e);
throw new RuntimeException(e);
```

per the Guava article since we don't know if `e` is checked or unchecked, and it's probably good practice to avoid wrapping exceptions when it's not necessary.

However, I looked at the code in the `try` block a bit more closely, and it appears the only checked exception it throws is `IOException`, and the only unchecked exceptions it throws are due to a true programmer error (e.g. initializing the `connectTimeoutInMilliseconds` local variable to a negative value), which we want to propagate anyway.

So I agree with leaving this line as-is, but we should probably change the exception type in the `catch` to `IOException`.